numpy/random/mtrand/mtrand.pyx

```diff
@@ -4460,8 +4460,8 @@ cdef class RandomState:
 
         A loaded dice is more likely to land on number 6:
 
-        >>> np.random.multinomial(100, [1/7.]*5)
-        array([13, 16, 13, 16, 42])
+        >>> np.random.multinomial(100, [1/7.]*6)
```

Comment:
The examples should primarily show you how to use it properly rather than all of the legal but confusing ways to abuse the API. The convention is intended to avoid floating point problems under the assumption that you are giving it values that are already normalized to the best of your ability within floating point `eps`. Under no circumstances should you ever intentionally give it grossly unnormalized values.